scipy/special/_hypergeometric.pxd

```diff
@@ -25,3 +30,53 @@ cdef inline double hyperu(double a, double b, double x) nogil:
             return poch(1.0 - b + a, -a)
 
     return hypU_wrap(a, b, x)
+
+
+cdef inline double hyp1f1(double a, double b, double x) nogil:
+    if npy_isnan(a) or npy_isnan(b) or npy_isnan(x):
+        return NPY_NAN
+    if b <= 0 and b == floor(b):
+        return NPY_INFINITY
+    elif a == 0 or x == 0:
+        return 1
+    elif a == -1:
+        return 1 - x / b
+    elif a == b:
+        return exp(x)
+    elif a - b == 1:
+        return (1 + x / b) * exp(x)
+    elif a == 1 and b == 2:
+        return (exp(x) - 1) / x
+    elif a <= 0 and a == floor(a):
+        # For `a` a negative integer the series is finite.
+        return hyp1f1_series(a, b, x)
+
+    if b > 0 and (fabs(a) + 1) * fabs(x) < 0.9 * b:
+        # For the kth term of the series we are multiplying by
+        #
+        # t_k = (a + b) * x / ((b + k) * (k + 1))
```

Comment:
Typo here: `(a + k)` not `(a + b)` (it made me wonder what's going on...)

Reply:
Oops, sorry about that...